modules/understanding-upgrade-channels.adoc

```diff
@@ -50,7 +52,7 @@ for more build information.
 [discrete]
 == fast-{product-version} channel
 
-The `fast-{product-version}` channel is updated with new {product-version} versions as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
+The `fast-{product-version}` channel is updated with new {product-version} and previous minor versions as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
```

Comment:
Might be misunderstanding, but the first sentence read a little funny to me. Could it be something like:

```
The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release.
```

?

Reply:
Agreed! Just made that update :+1: . Thanks!